src/App.js

```diff
@@ -7,7 +7,8 @@ import { getFirestore } from 'firebase/firestore';
 import AppContext from './components/AppContext';
 import config from './config';
 
-function App() {
+
+export default function App() {
```

Comment:
Why change this here and not everywhere else?

Reply:
Exports are agreed to stay at bottom.